### ISSUE TITLE
labels Strategies

### ISSUE THREAD

**Issue by yianni-ververis** [qlik-oss/picasso.js#52]
### Description

There is only one strategy for labels, "bar" and it only applies to bar charts. This means that any other chart, like a pie, you cannot have labels

### Steps to Reproduce

1. Add Labels component into a pie component
2. Set strategy type "bar"

##### Expected behavior

See labels on "path" element

##### Actual behavior

If I have time, I will try and add it under `src/core/chart-components/labels/strategies.js`

### Versions

* picasso.js: 
* Browser: 
* [Other relevant versions]

**Comment by yianni-ververis** [comment]
It needs one for circle too, for scatter plots etc

**Comment by cbt1** [comment]
Agree, we need to add strategies for other types than "bars". An FYI though, the bar strategy should "work" with any shape as the position is based on the bounding rectangle of a shape.
